lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
`bin/rspec` now works consistently when launched from any ruby and startup time is **much much** better that using `bin/logstash rspec` and I included it in the package. Let me know how you guys feel about that.